CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ All notable changes to the Refract project will be documented in this file.
 
 - Moved Compact Refract out of base specification
 - Removes the concept of namespaces and uses profiles instead
+- `Element Pointer` has become an element itself instead of an enum.
```

Comment:
There is no tag for 0.7.0 so there is no point of reference for that is in 0.7.0.